tensorboard/plugins/graph/tf_graph_common/op.ts

```diff
@@ -364,10 +471,17 @@ module tf.graph.op {
       'Switch',
       // Ops below are inserted by the compiler.
       '_Arg',
+      '_ArrayToList',
+      '_FusedBatchNormEx',
+      '_ListToArray',
       '_ParallelConcatUpdate',
+      '_RecvTPUEmbeddingActivations',
+      '_RecvTPUEmbeddingDeduplicationData',
       '_Retval',
       '_TPUCompile',
       '_TPUExecute',
+      '_SendTPUEmbeddingGradients',
```

Comment:
nit: should `_Send...` should be alphabetcally sorted before L481?

Reply:
Done